index.js

```diff
@@ -1,57 +1,115 @@
+	const info = {
+		query: {
+			match: {
+				markdown: {
+					query: req.body.query,
+				},
+			},
+		},
+	};
+
+	const url = `${URI}/${INDEX}/_search?pretty`;
+	const elasticRes = await (
+		await fetch(url, {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/x-ndjson',
+			},
+			body: JSON.stringify(info),
+		})
+	).json();
+
+	// error catch
+	if (!elasticRes)
+		return res.status(500).json({ error: 'Internal server error' });
+	if (elasticRes.timed_out) return res.status(400).json({ error: 'Timeout' });
+	if (elasticRes.hits.hits.length < 0)
+		return res.status(400).json({ error: 'No Result' });
 
-  const info = {
-    query: {
-      match: {
-        markdown: {
-          query: req.body.query,
-        }
-      }
-    }
-  }
-
-  const url = `${URI}/${INDEX}/_search?pretty`
-  const elasticRes = await (await fetch(url, {
-    method: "POST",
-    headers: {
-      'Content-Type': 'application/x-ndjson'
-    },
-    body: JSON.stringify(info)
-  })).json();
-  
-  // error catch
-  if (!elasticRes) return res.status(500).json({ error: 'Internal server error' });
-  if (elasticRes.timed_out) return res.status(400).json({ error: 'Timeout' });
-  if (elasticRes.hits.hits.length < 0) return res.status(400).json({ error: 'No Result' });
-
-  res.status(200).json({
-    data: {
-      totalHits: elasticRes.hits.total.value,
-      hits: elasticRes.hits.hits.map((hit) => ({
-        code: hit._source.code,
-        score: hit._score,
-      }))
-    }
-  });
+	res.status(200).json({
+		data: {
+			totalHits: elasticRes.hits.total.value,
+			hits: elasticRes.hits.hits.map((hit) => ({
+				code: hit._source.code,
+				score: hit._score,
+			})),
+		},
+	});
 });
 
 app.get('/api/v1/ir', (req, res) => {
-  res.status(200).json({ message: 'Typhon: Hello World' });
+	res.status(200).json({ message: 'Typhon: Hello World' });
+});
+
+app.get('/api/v1/ty/all', async (req, res) => {
+	const result = await weaviateClient.graphql
+		.get()
+		.withClassName('Code')
+		.withFields('code _additional { id }')
+		.do();
+
+	const code_results = result['data']['Get']['Code'].map((item) => ({
+		code: item['code'],
+		score: item['_additional']['certainty'],
+		id: item['_additional']['id'],
+	}));
+
+	data = {
+		totalHits: code_results.length,
+		hits: code_results,
+	};
+	res.status(200).json(data);
+});
+
+app.post('/api/v1/ty', async (req, res) => {
+	const nearText = {
+		concepts: [req.body.query],
+	};
+
+	const fetchedResult = await weaviateClient.graphql
+		.get()
+		.withClassName('Code')
+		.withNearText(nearText)
+		.withFields('code _additional { id certainty }')
+		.do();
+
+	const code_results = fetchedResult['data']['Get']['Code'].map((item) => ({
+		code: item['code'],
+		score: item['_additional']['certainty'],
+		id: item['_additional']['id'],
+	}));
+
+	data = {
+		totalHits: code_results.length,
+		hits: code_results,
+	};
+	res.status(200).json(data);
+});
+
+app.get('/api/v1/ty', async (req, res) => {
```

Comment:
```suggestion
app.get('/api/v1/ml', async (req, res) => {
```
use `/api/v1/ml` instead of `api/v1/ty`